src/view/sidebar.inc

```diff
@@ -48,6 +56,23 @@
       {% endif %}
       </ul>
     </div>
+
+
+    <div class="tab-content">
+      <div role="tabpanel" class="tab-pane active" id="tab-alpha">
+      </div>
+      <div role="tabpanel" class="tab-pane" id="tab-hierarchy">
+        <p>bbb</p>
+      </div>
+      <div role="tabpanel" class="tab-pane" id="tab-groups">
+        <p>ccc</p>
+      </div>
+      <div role="tabpanel" class="tab-pane" id="tab-changes">
+        <p>ddd</p>
+      </div>
+    </div>
+
+
     {% if letters %}
```

Comment:
Remove this `if letters` block (lines 51-64) as the functionality was reimplemented in Vue